### PR TITLE
Added bursts (Rivtech guns) [WIP]

### PR DESCRIPTION
**Pull request by BorkBorkGoesTheCode**
From #17309

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
Do we have a ```SIMULTANEOUS```` flag for burst fire? That's the sort of burst the scout rifle is supposed to have, same as the AN-94.

**Comment by BorkBorkGoesTheCode** [comment]
@mugling said SIMULTANEOUS wasn't finished yet. When it is finished, I will open a new PR to add it.
BTW, did you know ci.narc.ro says all the windows builds are broken?

**Comment by Rivet-the-Zombie** [comment]
Looks like it's just the Win/Curses versions, and no I did not.

**Comment by BorkBorkGoesTheCode** [comment]
Curses foiled again :) (I could not resist the urge)

**Comment by mugling** [comment]
If you're going to rework burst fire for a group of a grouping of guns then it's reasonable to add `SIMULTANEOUS` at the same time as a statement of intent as part of a larger PR.

File: data/json/items/gun/8x40mm.json [hunk 1]
```diff
@@ -201,6 +213,11 @@
     "durability": 9,
     "barrel_length": 2,
     "burst": 10,
```
Comment (mugling) [inline data/json/items/gun/8x40mm.json]: Did you test this a I don't think you can specify both `burst` and `modes`

**Comment by BorkBorkGoesTheCode** [comment]
Temporarily closed

**Pull request closed without merging**